nolan-swerve-2023/src/main/java/frc/robot/Constants.java

```diff
@@ -57,4 +61,26 @@ public final class Constants {
     public static final int BACK_RIGHT_MODULE_STEER_MOTOR = 18; 
     public static final int BACK_RIGHT_MODULE_STEER_ENCODER = 22; 
     public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(9.45); // FIXME Measure and set back right steer offset
+
+	public static final double DEADBAND = 0.1;
+
+    public static SwerveDriveSubsystem.Configuration GET_SWERVE_DRIVE_CONFIG() {
+        SwerveDriveSubsystem.Configuration config = new SwerveDriveSubsystem.Configuration();
+
+        config.m_trackwidthMeters = .60325;
+        config.m_wheelbaseMeters = .62865;
+
+        config.m_maxVoltage = 10.0;
+        config.m_maxVelocityMetersPerSecond = 6380.0 / 60.0 *
+                SdsModuleConfigurations.MK4I_L2.getDriveReduction() *
+                SdsModuleConfigurations.MK4I_L2.getWheelDiameter() * Math.PI;
+        config.m_maxAngularVelocityRadiansPerSecond = config.m_maxVelocityMetersPerSecond /
+                Math.hypot(config.m_trackwidthMeters / 2.0, config.m_wheelbaseMeters / 2.0);
+
+        config.m_pXController = .56122;
```

Comment:
Either do the full controller or create configs for all three PID values. We may not need them at the moment, but we might in the future.